src/foam/nanos/theme/themes.jrl

```diff
@@ -3,11 +3,12 @@ p({
   "id":"79B87FC0-C622-417A-A14A-2A04DD348CBD",
   "spid":"foam",
   "name":"foam",
+  "registrationGroup":"foam",
   "appName":"FOAM",
   "description":"The default FOAM theme.",
   "topNavigation":"foam.nanos.u2.navigation.ResponsiveTopNav",
   "defaultMenu": ["admin.data"],
-  "logo":"/images/foam_red.png",
+  "logo":"images/foam_red.png",
```

Comment:
This breaks the logo for me when I hit

http://localhost:8080/foam3/src/foam/nanos/controller/index.html

running from raw foam3 nanos

Reply:
Not sure how to solve this one for both parties. 
When are running/using just javascript foam? Meaning - if I understand the use case, maybe it will illicit a solution.

Reply:
While working on this I can't find foam_red.svg or foam_red.png anywhere in the repository.

Reply:
In the original clientloginservice work this was not working, but not the original path is working for all cases.

Reply:
I'm pretty sure /images/foam_red.png should work for all cases?  Doesn't the ImagesServlet serve all images from the file tree under /images/... ?